sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -863,6 +865,24 @@ public void createTransmissionTargetBeforeStart() {
         AppCenterLog.error(anyString(), contains("AppCenter is not configured"));
     }
 
+    @Test
+    public void triggerIntervalReturnFalse() {
+        Analytics analytics = Analytics.getInstance();
```

Comment:
This test is flawed in the logic.

You need to start before checking for min and max value.
As for checking for not started you need to pass a valid value to know the reason it returns false is because not started.